bounties/npm/shlv/1/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": 1,
+    "DisclosureDate": "2020-12-25",
+    "AffectedVersionRange": "*",
+    "Summary": "Prototype Pollution",
+    "Contributor": {
+        "Discloser": "Gavin Matthew",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "shvl",
+        "URL": "https://www.npmjs.com/package/shvl",
+        "Downloads": "433132/month"
```

Comment:
Please remove `/month` :+1:

Reply:
I have deleted my huntr repository due to multiple commits while submitting my next vulnerability disclosure and I re-forked it again. Due to this I am unable to make the modification to this file. Can you please do the modification on behalf of me. Thank you very much in advance.